### PR TITLE
Cycle

### PR DESCRIPTION
**Pull request by Kartynnikk**

### CODE REVIEW

File: cycle.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+for (let n = 2; n <= 10; n++) {
+    if (n % 2 == 0) {
```
Comment (kathrynowy) [inline cycle.js:2]: rewrite condition without == 0

File: cycle.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+while (n <= 100 && n) {
+    alert(n);
+}
+
+let n = 10;
+NextPrime:
+    for (let a = 2; a <= n; a++) {
+        for (let b = 2; b < a; b++) {
+            if (a % b == 0) continue NextPrime;
```
Comment (kathrynowy) [inline cycle.js:25]: same

File: cycle.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+let n;
+do {
+    n = prompt("Введите число больше 100", 0);
+}
+while (n <= 100 && n) {
+    alert(n);
+}
+
+let n = 10;
+NextPrime:
```
Comment (kathrynowy) [inline cycle.js:22]: do we name variables like that?

Comment (kathrynowy) [reply cycle.js:22]: this task is taken from the internet.

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: cycle.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cycle</title>
+</head>
+
+<body>
+    <script src="cycle.js"></script>
+
+</body>
+
+</html>
```
Comment (kathrynowy) [inline cycle.html:16]: same here

File: cycle.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+    n++
+}
+
+for (let n = 11; n <= 33; n++) {
+    alert(n);
+}
+
+
+for (let n = 0; n <= 100; n++) {
+    if (n % 2 == 0) {
```
Comment (kathrynowy) [inline cycle.js:62]: remove == 0

you know how to write it in a proper way

File: cycle.js [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+for (let n = 0; n <= 100; n++) {
+    sum += n;
+}
+alert(sum);
+
+
+let a = 1000,
+    n = 0;
+while (a >= 50) {
+    a = a / 2;
```
Comment (kathrynowy) [inline cycle.js:78]: /=

File: cycle.js [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+    a + 2;
+}
+
+for (let a = 1; a <= 10; a++) {
+    console.log(a * 7);
+}
+
+for (let a = 1000; a <= 2000; a++) {
+    console.log("&" + "#" + a);
+}
```
Comment (kathrynowy) [inline cycle.js:156]: eof

File: prch.html [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    for (let a = 1000; a <= 2000; a++) {
+        console.log("&#" + a);
+    }
+</script>
+
+<body>
+
+</body>
+
+</html>
```
Comment (kathrynowy) [inline prch.html:20]: check EOF everywhere please :)

**Review by kathrynowy — CHANGES_REQUESTED** [review]

File: cycle.js (hunk 1 above)
Comment (Kartynnikk) [reply cycle.js]: fixed

File: cycle.js (hunk 2 above)
Comment (Kartynnikk) [reply cycle.js]: fixed

File: cycle.js (hunk 3 above)
Comment (Kartynnikk) [reply cycle.js]: fixed

File: cycle.html (hunk 4 above)
Comment (Kartynnikk) [reply cycle.html:16]: fixed

File: cycle.js (hunk 5 above)
Comment (Kartynnikk) [reply cycle.js:62]: fixed

File: cycle.js (hunk 6 above)
Comment (Kartynnikk) [reply cycle.js:78]: fixed

File: cycle.js (hunk 7 above)
Comment (Kartynnikk) [reply cycle.js:156]: fixed

File: prch.html (hunk 8 above)
Comment (Kartynnikk) [reply prch.html:20]: fixed

**Review by kathrynowy — APPROVED** [review]

**Pull request merged**